setup.py

```diff
@@ -1,6 +1,7 @@
-from pathlib    import Path
-import setuptools
 import sys
+from pathlib import Path
+
+import setuptools
 
```

Comment:
Looks like you have some sort of auto formatter on save? These changes aren't relevant (also some in file.py)

Reply:
Yeah they are isort - a nice logical formatter for imports 🙃 

The sorting is: a) by group, first stdlib, then external dependencies, than internal modules, b) within each group alphabetical. but I see your point we should do this systematically across the board

Reply:
Undid the formatting changes